pkg/apis/networking/v1alpha1/ingress_lifecycle.go

```diff
@@ -71,6 +71,14 @@ func (is *IngressStatus) MarkLoadBalancerReady(lbs []LoadBalancerIngressStatus,
 	ingressCondSet.Manage(is).MarkTrue(IngressConditionLoadBalancerReady)
 }
 
+// MarkLoadBalancerPending marks the "IngressConditionLoadBalancerReady" condition to unknown to
+// reflect that the load balancer is not ready yet.
+func (is *IngressStatus) MarkLoadBalancerPending() {
+	reason := "Uninitialized"
```

Comment:
those are consts, and do we really need them?

Reply:
Can you clarify what you mean?
This is adapted from what we do [here](https://github.com/knative/serving/blob/master/pkg/apis/serving/v1alpha1/service_lifecycle.go#L84) or [here](https://github.com/knative/serving/blob/master/pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go#L51) for example.

Reply:
I mean why do we need this variable. Both examples your cited have no variables.